modules/apps/data-engine/data-engine-taglib/src/main/resources/META-INF/resources/data_layout_builder/js/components/rules/editor/Conditions.es.js

```diff
@@ -130,8 +152,14 @@ function FieldLeft({fields, left, onChange}) {
 					},
 				]}
 				onChange={onChange}
+				onSelectionChange={(itemKey) => {
+					setFieldLeftSelectedKey(itemKey);
```

Comment:
Hi @rebsilva, I'm not sure if I got your point, but I made the tests e it's working fine using the keyboard.

[Rules-using-keyboard.webm](https://github.com/liferay-forms/liferay-portal/assets/26491796/e71ce81a-b643-4f70-a6f2-014fc8ecbff8)

Reply:
great @alinedoleron!! thanks for testing, i think in this case works because you were setting as `undefined` as we discussed in the previous comment, so its not the same as the other bugs i've showed here. thanks a lot! :heart: